html/mycakeapp/src/Controller/MainController.php

```diff
@@ -108,4 +110,36 @@ public function top()
       ->toArray();
     $this->set(compact('slideshowPictures', 'moviePictures', 'discountPictures'));
   }
+  public function price()
+  {
+    $this->viewBuilder()->setLayout('no-frame');
+    $today = Time::now()->format('Y-m-d H:i:s');
+    $fees = $this->Fees->find()->select(['name', 'fee'])
+      ->where([
+        'is_deleted' => 0,
+        'started_at <=' => $today,
+        'finished_at >=' => $today,
+      ])
```

Comment:
料金適用日時がnullの場合もありますので、その対応をお願いします！
例↓
```suggestion
      ->where([
        'is_deleted' => 0,
        'started_at <=' => $today,
        'OR' => [['finished_at >=' => $today], ['finished_at IS NULL']],
      ])
```

Reply:
おっしゃる通りですね、失礼しました。
修正します！！